src/components/victory-candlestick/victory-candlestick.js

```diff
@@ -12,22 +12,24 @@ const fallbackProps = {
     height: 300,
     width: 450,
     candleColors: {
-      positive: "green",
-      negative: "red"
+      positive: "#ffffff",
+      negative: "#252525"
     }
   },
   style: {
     data: {
       opacity: 1,
+      stroke: "#252525",
```

Comment:
@ebrillhart having a stroke color defined here means that the candle wick will never default to the fill color. If you want that behavior by default, I think removing this will do the trick.